opentreemap/importer/models/base.py

```diff
@@ -47,6 +48,9 @@ class Meta:
 
     status = models.IntegerField(default=PENDING_VERIFICATION)
 
+    last_processed_at = models.DateTimeField(null=True, blank=True)
+    is_lost = models.BooleanField(default=False)
```

Comment:
Like all new fields these days this should be null=True with a TODO (or an unmerged PR) to remove null=True.